.github/workflows/benchbringup.yml

```diff
@@ -0,0 +1,61 @@
+name: Run a Benchmark
+on:
+  workflow_dispatch:
+    inputs:
+      timeout:
+        description: 'Timeout for each test case (in seconds)'
+        required: true
+        default: '300'
+      options:
+        description: 'Command-line flags to pass via ESBMC_OPTS'
+        required: false
+        default: '-I /__w/esbmc/esbmc/src/cpp/library'
```

Comment:
`-I` is most likely not the right position in the include-dir list: `-I` puts it pretty much at the beginning of the search paths, while it should definitely not override user-given `-I`. `-isystem` would be the correct place, I believe. See https://github.com/esbmc/esbmc/blob/b6aac2b30bce8039426f994ca7e970b9f36e1386/src/clang-c-frontend/clang_c_language.cpp#L38-L43 This brings me to another question: Shouldn't adding this include path be handled by the `clang_cpp_language` in a similar manner?

Reply:
> Shouldn't adding this include path be handled by the clang_cpp_language in a similar manner?

It would be nice to have a coherent way to include operational model for both C and C++. However, this will be out of the scope as this PR just implements what's necessary to collect logs and get stats about the error signatures to guide our development. 

For the time being, we are still following the way old ESBMC++ includes these operational models using `-I` (in such a way unfortunately we have to ask the user to be careful with the `-I` order since the order matters). e.g.: 
```
cmd: "-I /home/ks/ESBMC_Project/esbmc/src/cpp/library -I ../../lib_dummy_2 -I ../../lib_dummy_1"

#include <...> search starts here:
 /home/ks/ESBMC_Project/esbmc/src/cpp/library
 ../../lib_dummy_2
 ../../lib_dummy_1
 /tmp/esbmc.78af-7fd4-f2b8/headers
 /tmp/esbmc-headers-980a-77a4-3e54
 /../lib/gcc/x86_64-linux-gnu/9/../../../../include/c++/9
 /../lib/gcc/x86_64-linux-gnu/9/../../../../include/x86_64-linux-gnu/c++/9
 /../lib/gcc/x86_64-linux-gnu/9/../../../../include/c++/9/backward
 /usr/local/include
 /usr/include/x86_64-linux-gnu
 /usr/include
End of search list.
```

Reply:
As you agree that it's wrong, could you add a TODO-like comment to the `-I` in order to remind us that it's not as intended, yet?

Reply:
Done

Reply:
Not sure what it's doing there, but better some TODO than none TODO. ;)